src/components/PeopleFilters.tsx

```diff
@@ -84,6 +103,7 @@ export const PeopleFilters = () => {
         <a
           className="button is-link is-outlined is-fullwidth"
           href="#/people"
+          onClick={() => setQuery('')}
```

Comment:
Such functions are better placed in separate handlers:
```
function handleResetFilters() {
  setQuery('')
}
...
onClick={handleResetFilters}
```
This improves the readability and understandability of the code.